books/free-programming-books-langs.md

```diff
@@ -2592,6 +2593,11 @@ Books on general-purpose programming that don't focus on a specific language are
 * [V Documentation](https://github.com/vlang/v/blob/HEAD/doc/docs.md) - vlang.io (Markdown)
 
 
+### Vala
+
+* [Vala Documentation](https://wiki.gnome.org/Projects/Vala/Documentation)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
I do have a couple more resources for this, but they're not in the repo yet